app/views/clients/_homelessness.haml

```diff
@@ -33,6 +33,9 @@
     %li.c-detailed-list__item
       %span.c-detailed-list__label Last Homeless Contact
       %span.c-detailed-list__value= @client.calculated_last_homeless_night.try(:strftime, I18n.t('date.formats.default'))
+    %li.c-detailed-list__item
+      %span.c-detailed-list__label Encampment Decomissioned
+      %span.c-detailed-list__value= boolean_icon @client.encampment_decomissioned
     %li.c-detailed-list__item
       %span.c-detailed-list__label Chronically Homeless
       %span.c-detailed-list__value= boolean_icon @client.chronic_homeless
```

Comment:
I added Encampment Decomissioned to the "Homelessness" section of the client page. Is this the right place to put it @awisema @eanders? 
Also, should we flag this behind something? For most installations, it will always be false.

Reply:
Sounds good.  I think it's okay to leave it up for everyone for now.